### PR TITLE
Added Polar coordinates related features

### PR DESCRIPTION
**Pull request by AdvaitDhingra**
This was not in any issue, but I thought this would be very helpful. This way, you don't need to convert theta and phi into Cartesian vectors manually, and vice-versa.

I tested it on Ubuntu 20.04 and it works flawlessly.

### CODE REVIEW

File: math/physics/inc/TVector3.h [hunk 1]
```diff
@@ -34,6 +34,9 @@ class TVector3 : public TObject {
    TVector3(const Float_t *);
    // Constructors from an array
 
+   TVector3(Double_t phi, Double_t theta);
```
Comment (Axel-Naumann) [inline math/physics/inc/TVector3.h:37]: What's Pt here?

Comment (AdvaitDhingra) [reply math/physics/inc/TVector3.h:37]: Sorry, what is pt? 😅

**Comment by oshadura** [comment]
@phsft-bot  build!

File: math/physics/inc/TVector3.h (hunk 1 above)
Comment (AdvaitDhingra) [reply math/physics/inc/TVector3.h:37]: Here, the two angles phi and theta allow you to make a transformation vector. If you have a point and two angles theta and phi, for example, you could use polar coordinates to get a transformation vector to calculate the trajectory. This function allows you to create a Cartesian Transformation Vector from two angles.

**Comment by AdvaitDhingra** [comment]
Nevermind, I just found Polar coordinate related features in Polar3D. Sorry for wasting your time.

**Pull request closed without merging**